### PR TITLE
Update onboarding experiments

### PR DESCRIPTION
**Pull request by liuche**
Switchboard changes to match new local onboarding experiments from bug 1238780 [1].

[1] https://bugzilla.mozilla.org/show_bug.cgi?id=1238780

### CODE REVIEW

**Comment by leibovic** [comment]
This raises the question of how we should manage this configuration across product versions... this config is designed to work with the code that will be in Fx46, but not in Fx45 and beyond.

Maybe we should create new names for these experiments, rather than re-using "onboarding-a" and "onboarding-b". Given the recent changes in bug 1241591, we should update the config so that 100% of users are getting "onboarding-a".

Once we get support for matching based on app ID, we can turn experiments on/off for different release channels, but that doesn't exactly address this issue, where we want an experiment to ride the trains, but it will only be available in a certain version.

**Comment by liuche** [comment]
This adds versions to these onboarding experiments. At the moment, it's 46-49, because I expect we'll turn this experiment off by then and I didn't want to write a more complicated regex, but we could also amend this to be 46-49|50+ easily.

Tested this regex locally in python with some version strings.

**Comment by leibovic** [comment]
I still think we should change the names of the experiments to avoid a foot-gun. If we're doing analysis of experiments with retention, we'd need to include versions numbers in that analysis as well, and that seems like an unnecessary extra level of complexity.

Could we update the names of the experiments here and in the client?

File: experiments.json [hunk 1]
```diff
@@ -1,15 +1,28 @@
 {
-  "onboarding-a": {
```
Comment (leibovic) [inline experiments.json:2]: Also, we should still keep an "onboarding-a" set to 100% of users, for versions 43-45.

**Comment by liuche** [comment]
Ugh, github - this branch didn't seem to pick up commits I'd made and pushed, so closing this and opening a new pull request with the new commits.

**Pull request closed without merging**